pekko-connectors-sample-http-csv-to-kafka/step_001_http_request/src/main/scala/samples/Main.scala

```diff
@@ -4,21 +4,21 @@
 
 package samples
 
-import akka.Done
-import akka.actor._
-import akka.http.scaladsl._
-import akka.http.scaladsl.model.StatusCodes._
-import akka.http.scaladsl.model.headers.Accept
-import akka.http.scaladsl.model.{ HttpRequest, HttpResponse, MediaRanges }
-import akka.stream._
-import akka.stream.scaladsl.{ Sink, Source }
-import akka.util.ByteString
+import org.apache.pekko.Done
```

Comment:
@reypader Pekko uses a root nested imports style, that is

```scala
import org.apache.pekko
import pekko.Done
import pekko.actor._
etc etc
```

Would you be able to update the PR to this style for all `.scala` source files?

Reply:
I've been trying to play around with my formatters and scalafmt. I copied over scalafmt from `pekko-connectors` project even. No luck. I can't seem to find formatter config that would automatically do this for me. Could you show me how?

Reply:
It looks like you have a similar thread on this specific topic https://github.com/apache/incubator-pekko/issues/414 

Since this project is separate from `incubator-pekko` maybe it would be easier to enable scalafix here?

Reply:
Regarding scalafmt, this can be done later. For now I would just manually do the change I suggested